test/framework/docker_logcollector.go

```diff
@@ -45,6 +46,22 @@ func machineContainerName(cluster, machine string) string {
 
 func (k DockerLogCollector) CollectMachineLog(ctx context.Context, managementClusterClient client.Client, m *clusterv1.Machine, outputPath string) error {
 	containerName := machineContainerName(m.Spec.ClusterName, m.Name)
+	return k.collectLogsFromNode(outputPath, containerName)
+}
+
+func (k DockerLogCollector) CollectMachinePoolLog(ctx context.Context, managementClusterClient client.Client, m *expv1.MachinePool, outputPath string) error {
+	var reterr error
+	for _, instance := range m.Status.NodeRefs {
```

Comment:
The assumption that docker machine and node have the same name is kind of brittle, at least because this is not documented.
What adding commenting into machineContainerName where we take note that docker machine and node should have the same name because we rely on this for log collection?

Reply:
I assume there is no way to not rely on this?